webapp/templates/partials/scoreboard_table.html.twig

```diff
@@ -413,9 +413,13 @@
     {% for color,i in backgroundColors %}
         {% set colorClass = color | replace({"#": "_"}) %}
 
-        .cl{{ colorClass }} {
-            background-color: {{ color }};
-        }
+        {#html[data-bs-theme="dark"] .cl{{ colorClass }} {#}
```

Comment:
TODO: solve how to do this for dark mode

Reply:
solved https://github.com/delta-cs/seminar-domjudge/pull/154/commits/a8f0ceabdb190ab06f547801669898ac792c94e6